Go/Snippets/go-make.sublime-snippet

```diff
@@ -0,0 +1,6 @@
+<snippet>
+    <content><![CDATA[make(${1:type}, 0)]]></content>
```

Comment:
This seems more like a completion than a snippet to me. Same for `map`.

Reply:
is it okay to include a sublime-completions file in the default packages?